packages/studio-base/src/players/FoxgloveWebSocketPlayer/index.ts

```diff
@@ -320,11 +305,11 @@ export default class FoxgloveWebSocketPlayer implements Player {
 
       const time = fromNanoSec(timestamp);
       if (this._clockTime != undefined && isLessThan(time, this._clockTime)) {
-        ++this._lastSeekTime;
+        this._lastSeekTime = time.sec;
```

Comment:
@jhurliman maybe better to use `time.nsec` here? Otherwise multiple seeks in the same second might not be detected?

```suggestion
        this._lastSeekTime = time.nsec;
```

Reply:
`toNanoSec(time)` and make _lastSeekTime a bigint

Reply:
I don't think this change will behave as intended. If the clock time resets from >0 to =0 multiple times, the `_lastSeekTime` will not change, which means that panels throughout the app will not be reset. If the concern is with the name of the variable, I'd suggest renaming the variable instead.

Reply:
That's a valid point, I will rename the variable and revert back to incrementing it on every seek